extensions/kafka-client/deployment/src/main/java/io/quarkus/kafka/client/deployment/KafkaProcessor.java

```diff
@@ -91,10 +90,7 @@
 import io.quarkus.dev.spi.DevModeType;
 import io.quarkus.devconsole.spi.DevConsoleRouteBuildItem;
 import io.quarkus.devconsole.spi.DevConsoleWebjarBuildItem;
-import io.quarkus.kafka.client.runtime.KafkaAdminClient;
-import io.quarkus.kafka.client.runtime.KafkaBindingConverter;
-import io.quarkus.kafka.client.runtime.KafkaRecorder;
-import io.quarkus.kafka.client.runtime.KafkaRuntimeConfigProducer;
+import io.quarkus.kafka.client.runtime.*;
```

Comment:
😄 I had wanted to comment on that as well when I initially reviewed the PR. However, since I didn't find anything about imports in your coding guidelines I thought it must be ok for the team.